src/frontend/src/pages/FinancePage/RefundsSection.tsx

```diff
@@ -112,16 +112,28 @@ const Refunds = ({ userReimbursementRequests, allReimbursementRequests }: Refund
               </TableRow>
             </TableHead>
             <TableBody>
-              {rows.map((row, index) => (
-                <TableRow
-                  key={`${row.date}-$${row.amount}-${index}`}
-                  sx={{ '&:last-child td, &:last-child th': { border: 0 } }}
-                >
-                  <TableCell align="center">{datePipe(row.date)}</TableCell>
-                  <TableCell align="center">{centsToDollar(row.amount)}</TableCell>
-                  {tabValue === 1 && <TableCell align="center">{fullNamePipe(row.recipient)}</TableCell>}
-                </TableRow>
-              ))}
+              {rows
```

Comment:
you should sort the rows above where they're defined. Also you should be able to do this in less lines. a.date is already a date so you shouldn't need to cast it as one